sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/SparkThriftServerProtocolVersionsSuite.scala

```diff
@@ -148,49 +148,88 @@ class SparkThriftServerProtocolVersionsSuite extends HiveThriftJdbcTest {
       testExecuteStatementWithProtocolVersion(version, "SELECT cast(1 as byte)") { rs =>
         assert(rs.next())
         assert(rs.getByte(1) === 1.toByte)
+        val metaData = rs.getMetaData
+        assert(metaData.getColumnName(1) === "CAST(1 AS TINYINT)")
+        assert(metaData.getColumnType(1) === java.sql.Types.TINYINT)
```

Comment:
We don't need to check `getColumnTypeName`?

Reply:
thanks for the suggestions~